packages/protocol/contracts/verifiers/PseZkVerifier.sol

```diff
@@ -144,7 +144,6 @@ contract PseZkVerifier is EssentialContract, IVerifier {
             abi.encode(
                 tran.parentHash,
                 tran.blockHash,
-                tran.signalRoot,
```

Comment:
We need to verify the value is correct in the zk proof, so `tran.stateRoot` should still be here.

Reply:
Good catch